### DISCUSSION TITLE
Settings cmdheight=0 introduces "Press ENTER or type command to continue"

### DISCUSSION THREAD

**Discussion by marlun** [tpope/vim-fugitive#2061] [General]
If I have cmdheight=1 and enter cc in fugitive status window I can start to enter the commit message immediately but if I set cmdheight=0 to hide the command line I need to press ENTER before I continue. As I understand it it's because vim wants to show something that doesn't fit on the command line but it's not showing anything when cmdheight=1 either. Can I get around having to click ENTER even when the command line is hidden?

**Comment by tpope** [comment]
It's probably [this `echo ""`](https://github.com/tpope/vim-fugitive/blob/dd8107cabf5fe85df94d5eedcae52415e543f208/autoload/fugitive.vim#L3858), which is a pre-requisite of actually displaying output with [`echon`](https://github.com/tpope/vim-fugitive/blob/dd8107cabf5fe85df94d5eedcae52415e543f208/autoload/fugitive.vim#L3498). Conceivably we could restructure to make it lazy to handle this, but it probably makes more sense to handle it on the Neovim side.

**Reply by justinmk** [reply]
Yes, cmdheight=0 is currently experimental and Nvim needs to work out the bugs.
